sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -2548,6 +2548,14 @@ pi_result cuda_piKernelGetGroupInfo(pi_kernel kernel, pi_device device,
       return getInfo(param_value_size, param_value, param_value_size_ret,
                      pi_uint64(bytes));
     }
+    case PI_KERNEL_GROUP_INFO_NUM_REGS: {
+      int bytes = 0;
```

Comment:
Minor comment, but a name like `numRegs` or similar would be more fitting. The ones that call their local variable `bytes` are querying data sizes, whereas this one doesn't.